pallets/runtime/tests/src/staking/mod.rs

```diff
@@ -37,6 +37,33 @@ macro_rules! assert_session_era {
     };
 }
 
+macro_rules! assert_present_entity {
+    ($acc_id:expr) => {
+        assert_eq!(
+            Staking::permissioned_entities(Identity::get_identity($acc_id).unwrap()),
+            true
+        );
+    };
+}
+
+macro_rules! assert_absent_entity {
+    ($acc_id:expr) => {
+        assert_eq!(
+            Staking::permissioned_entities(Identity::get_identity($acc_id).unwrap()),
+            false
+        );
+    };
+}
+
+macro_rules! assert_add_permissioned_validator {
+    ($acc_id:expr) => {
+        assert_ok!(Staking::add_permissioned_validator_entity(
+            frame_system::RawOrigin::Root.into(),
+            Identity::get_identity($acc_id).unwrap()
+        ));
+    };
+}
```

Comment:
These don't need to be macros; they can be functions using `#[track_caller]`.

Reply:
Is there any performance or security issue?

Reply:
Well compile time perf; its less code to typecheck and stuff, but also macros give you better type checking errors, etc.